token-lending/js/src/util/layout.ts

```diff
@@ -79,7 +79,7 @@ export const bigInt =
 
         bigIntLayout.decode = (buffer: Buffer, offset: number) => {
             const src = decode(buffer, offset);
-            return toBigIntLE(src);
+            return toBigIntLE(src as Buffer);
```

Comment:
Is this still required after the CLI tsconfig change? Mostly wondering if I got something with the types wrong originally.

Reply:
Yep, still getting the error. The full version is:
```
/Users/ilmoi/.nvm/versions/node/v16.1.0/bin/node /Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/dist/bin.js /Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/cli/main.ts
/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:693
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
../src/util/layout.ts:82:31 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'Buffer'.
  Type '{}' is missing the following properties from type 'Buffer': write, toJSON, equals, compare, and 78 more.

82             return toBigIntLE(src);
                                 ~~~

    at createTSError (/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:693:12)
    at reportTSError (/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:697:19)
    at getOutput (/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:884:36)
    at Object.compile (/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:1186:30)
    at Module.m._compile (/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:1310:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/node_modules/ts-node/src/index.ts:1314:12)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19) {
  diagnosticText: "\x1B[96m../src/util/layout.ts\x1B[0m:\x1B[93m82\x1B[0m:\x1B[93m31\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS2345: \x1B[0mArgument of type 'unknown' is not assignable to parameter of type 'Buffer'.\n" +
    "  Type '{}' is missing the following properties from type 'Buffer': write, toJSON, equals, compare, and 78 more.\n" +
    '\n' +
    '\x1B[7m82\x1B[0m             return toBigIntLE(src);\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                              ~~~\x1B[0m\n',
  diagnosticCodes: [ 2345 ]
}

Process finished with exit code 1
```